src/js/views/forms/form/form_views.js

```diff
@@ -405,12 +405,30 @@ const UpdateView = View.extend({
 });
 
 const HistoryDroplist = Droplist.extend({
-  viewOptions: {
-    className: 'button-filter',
-    template: hbs`{{far "clock-rotate-left"}}{{formatDateTime updated_at "AT_TIME"}}{{far "angle-down"}}`,
+  viewOptions() {
+    return {
+      className: 'button-filter',
+      template: hbs`
+        {{far "clock-rotate-left"}}{{formatDateTime updated_at "AT_TIME"}} {{formatMessage (intlGet "forms.form.formViews.historyDroplistView.nameText") name=name}}{{far "angle-down"}}
+      `,
+      templateContext() {
+        return {
+          name: this.model.getEditorName(),
+        };
+      },
+    };
   },
-  picklistOptions: {
-    itemTemplate: hbs`{{formatDateTime updated_at "AT_TIME"}}`,
+  picklistOptions() {
+    return {
+      itemTemplate: hbs`
+        {{formatDateTime updated_at "AT_TIME"}}&nbsp;{{formatMessage (intlGet "forms.form.formViews.historyDroplistView.nameText") name=name}}
```

Comment:
The space between the `updated_at` date and `•` symbol was being removed for some reason. For example: `Aug 14, 2024 12:39 PM• By Nick Major`.

Seems to be because the picklist items are `display: flex`. I don't really like using `&nbsp;`, but it seemed simpler than messing with the picklist's css styles or adding an inline css style here.